src/main/java/net/dries007/tfc/objects/items/ItemsTFC.java

```diff
@@ -154,6 +158,8 @@ public static void registerItems(RegistryEvent.Register<Item> event)
         simpleItems.add(register(r, "wand", new ItemDebug(), CT_MISC));
         simpleItems.add(register(r, "mortar", new ItemMisc(Size.TINY, Weight.LIGHT, "mortar"), CT_MISC));
         register(r, "wooden_bucket", new ItemWoodenBucket(), CT_WOOD); //not a simple item, use a custom model
+        register(r, "blue_steel_bucket", new ItemMetalBucket(Metal.BLUE_STEEL, Metal.ItemType.BUCKET), CT_METAL); //not a simple item, use a custom model
+        register(r, "red_steel_bucket", new ItemMetalBucket(Metal.RED_STEEL, Metal.ItemType.BUCKET), CT_METAL); //not a simple item, use a custom model
```

Comment:
Why are these using a completely separate method of registration than the existing metal check? If the bucket is going to be an `ItemType`, it shouldn't exclude itself from the standard pattern of `metal/[type]/[metal]` registry names

Reply:
They're registering separate models because they have empty and fluid filled versions, and the simple registry of the rest of the Metal ItemTypes will not provide the model registration. We still want the metal information as well as the rest of the ItemMetal super class stuff, unless we want to reimplement it because the buckets are a special snowflake? Do we want to make a super class for buckets, and implement wooden and metal versions of them?

Reply:
I see the reason for separating them from the bulk model registration. It should still use the established naming convention though.

Reply:
Completed.